apps/etterlatte-trygdetid/src/main/kotlin/trygdetid/TrygdetidRoutes.kt

```diff
@@ -141,6 +141,26 @@ fun Route.trygdetid(
                 }
             }
 
+            post("/manuell/opprett") {
+                withBehandlingId(behandlingKlient) {
+                    logger.info("Oppretter trygdetid med overstyrt for behandling $behandlingId")
+                    trygdetidService.opprettOverstyrtBeregnetTrygdetid(behandlingId, brukerTokenInfo)
+                    call.respond(HttpStatusCode.OK)
+                }
+            }
+
+            post("/manuell/lagre") {
```

Comment:
Kunne man egentlig bare ha brukt den routen som allerede finnes for overstyring?

Reply:
Tenkte på det men det oppdaterer vel ikke status på behandling?

Reply:
Ah, det er kanskje en greie ja. Ser at route-navnet også måtte ha blitt endra da, så kanskje greit som det er ja.